models/lido_liquidity/kyberswap_ethereum/lido_liquidity_kyberswap_ethereum.sql

```diff
@@ -0,0 +1,270 @@
+{{ config(
+    alias = 'lido_liquidity_kyberswap_ethereum',
+    partition_by = ['time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'blockchain', 'time'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "lido",
+                                \'["ppclunghe", "gregshestakovlido"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2022-10-28' %} 
+
+with dates as (
+select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 hour)) as hour
+)
+ 
+, pools as (
+    select pool as address, 'Ethereum' as blockchain, 'Kyberswap' as project, swapFeeUnits/1000 as fee
+    from {{ source('kyber_ethereum', 'Elastic_Factory_evt_PoolCreated') }}
+    where token0 = lower('0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0')
+    or token1 = lower('0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0')
+)
+
+, tokens as (
+    select distinct token as address, pt.symbol, pt.decimals
+    from (
+    select token1 as token
+    from {{ source('kyber_ethereum', 'Elastic_Factory_evt_PoolCreated') }}
+    where token0 = lower('0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0')
+    union
+    select token0
+    from {{ source('kyber_ethereum', 'Elastic_Factory_evt_PoolCreated') }}
+    where token1 = lower('0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0')
+    union
+    select lower('0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0')
+    ) t
+    left join prices.tokens pt on t.token = pt.contract_address
+)
+
+, tokens_prices_daily AS (
+    SELECT distinct
+        DATE_TRUNC('day', minute) AS time,
+        contract_address as token,
+        avg(price) AS price
+    FROM {{ source('prices', 'usd') }}
+
+    {% if is_incremental() %}
+    WHERE date_trunc('day', minute) >= date_trunc("day", now() - interval '1 week')
+    and date_trunc('day', minute) < date_trunc('day', now())
+    and blockchain = 'ethereum'
```

Comment:
minor, but filters such as blockchain and contract_address below are in all runs, maybe you should move outside of the incremental blocks?